LAZY-TODO.txt

```diff
@@ -0,0 +1,8 @@
+[ ] repr (or not just repr) inconsistencies: (f*g).dx.dx.dx.dy  vs  (f*g).dx.dy
+[ ] lack of tests (lazy -> evaluation -> check expected "stencil")
+[ ] first_derivative, cross_derivative, etc, should return a Derivative object
```

Comment:
this is important.

Reply:
As said in my comments I disagree, I think we should keep these functions as finite_differences functions one for backward compatibility and two to be able to play with FD easily